contents/docs/integrate/cdp.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using PostHog with a CDP
+sidebarTitle: CDPs
+sidebar: Docs
+showTitle: true
+---
+
+## What is a CDP?
+
+A CDP is a Customer Data Platform. It is a platform that allows you to collect, and send customer data to other tools like product analytics (PostHog!), marketing automation tools, your CRM, data warehouses etc.
+
+The most common CDPs are [Segment](/docs/integrate/third-party/segment) and [Rudderstack](contents/docs/integrate/third-party/rudderstack), both of which work with PostHog.
+
+If you already have a Customer Data Platforms (CDP) setup or are thinking of using one this guide will help you decide how to integrate with PostHog.
+
+## Which PostHog CDP setup should I use?
+
+PostHog has many of the imports and exports of common (CDPs) built-in. This means there's two options for how you could integrate PostHog with a 3rd party CDP:
+
+1. Use PostHog as a CDP (recommended if you don't have a 3rd party CDP set up and we have the exports you need)
+2. Use a 3rd party CDP and integrate PostHog
+
+Which method you choose will depend on what your goals are, what you have already set up, and how much time and money you are willing to invest.
+
+Here's a decision tree that you might find handy:
+
+```mermaid
+graph TD
+A[Are you extensively using a 3rd party CDP?] --> |Yes| B[Option 2<br/>Use a 3rd party CDP and integrate PostHog]
+A --> |No| C[Are there any exports you need immediately that PostHog doesn't have yet?<br/>e.g. paid marketing platforms] --> |Yes| D[Option 2]
+C --> |No| E[Option 1<br/>Use PostHog as a CDP]
+```
+
+## Option 1: Use PostHog as a CDP
+
+In general, we'd highly recommend starting with PostHog as your CDP using the variety of imports and exports apps that we have. This is the easiest and least expensive way to get started.
+
+If you later need some extra destinations that we don't yet have you have several options: you can send your PostHog data to a 3rd party CDP ([see extra info here](#sending-posthog-data-to-a-3rd-party-cdp-for-extra-destinations)), create your own app, or fully transition to a 3rd party CDP (option 2). Every month we are adding more destinations.
```

Comment:
@lharries the biggest ask we get here is marketing automation/ad spend tracking.  Is it worthwhile linking to the roadmap from here so that people can register interest/vote on those types of exporters?

Reply:
Are they wanting to view the amount they spend across channels in PostHog or are they wanting to manage the tracking for the ads platforms (e.g. control the facebook pixel events via PostHog)?

Reply:
Yeah its more revenue tracking per channel that comes up than actually controlling things

Reply:
Regarding ad spend, this bit of context could be relevant. 

We (mostly me) tried to develop a "[google ads conversion plugin](https://github.com/PostHog/google-ad-conversions-plugin/issues/1)" back in the day. [This issue](https://github.com/PostHog/integrations-repository/issues/10) describes the work around it.

Our goal was to build a system that lets you attribute clicks to google ads (based on referrer info), but without the Google JS. The idea was to use their "offline conversion tracking" API to send a list with the click tracking IDs. This would tell Google which ads are being clicked on and which not, adjusting ad performance as relevant.

The trouble was that Google really wants you to use their system. Certain API endpoints are weirdly undocumented, and I didn't get any to return a correct response, despite trying to get the right access, over many different types of specialised google accounts.

I'd be happy if someone wanted to pick this up further, as to my knowledge, similar to mobile session recording, nobody else does it.

Reply:
Great to know @mariusandra - can imagine this would be v useful for the marketing and growth use-cases.

I think tracking the amount of adspend per channel is a separate simpler request I'll keep track of too